builtin.c

```diff
@@ -491,33 +492,87 @@ static const char builtin_not_recognised[] = "'%" SVf "' is not recognised as a
 
 static const struct BuiltinFuncDescriptor builtins[] = {
     /* constants */
-    { "true",   &XS_builtin_true,   &ck_builtin_const, BUILTIN_CONST_TRUE,  false },
-    { "false",  &XS_builtin_false,  &ck_builtin_const, BUILTIN_CONST_FALSE, false },
+    { "true",   5039, &XS_builtin_true,   &ck_builtin_const, BUILTIN_CONST_TRUE,  false },
+    { "false",  5039, &XS_builtin_false,  &ck_builtin_const, BUILTIN_CONST_FALSE, false },
 
     /* unary functions */
-    { "is_bool",    &XS_builtin_func1_scalar, &ck_builtin_func1, OP_IS_BOOL,    true  },
-    { "weaken",     &XS_builtin_func1_void,   &ck_builtin_func1, OP_WEAKEN,     false },
-    { "unweaken",   &XS_builtin_func1_void,   &ck_builtin_func1, OP_UNWEAKEN,   false },
-    { "is_weak",    &XS_builtin_func1_scalar, &ck_builtin_func1, OP_IS_WEAK,    false },
-    { "blessed",    &XS_builtin_func1_scalar, &ck_builtin_func1, OP_BLESSED,    false },
-    { "refaddr",    &XS_builtin_func1_scalar, &ck_builtin_func1, OP_REFADDR,    false },
-    { "reftype",    &XS_builtin_func1_scalar, &ck_builtin_func1, OP_REFTYPE,    false },
-    { "ceil",       &XS_builtin_func1_scalar, &ck_builtin_func1, OP_CEIL,       false },
-    { "floor",      &XS_builtin_func1_scalar, &ck_builtin_func1, OP_FLOOR,      false },
-    { "is_tainted", &XS_builtin_func1_scalar, &ck_builtin_func1, OP_IS_TAINTED, false },
-    { "trim",       &XS_builtin_trim,         &ck_builtin_func1, 0,             false },
-    { "stringify",  &XS_builtin_func1_scalar, &ck_builtin_func1, OP_STRINGIFY,  true },
-
-    { "created_as_string", &XS_builtin_created_as_string, &ck_builtin_func1, 0, true },
-    { "created_as_number", &XS_builtin_created_as_number, &ck_builtin_func1, 0, true },
+    { "is_bool",       0, &XS_builtin_func1_scalar, &ck_builtin_func1, OP_IS_BOOL,    true  },
```

Comment:
Instead of `0` and having to have an explicit check in `S_import_builtin_bundle`, could this be a named constant (e.g. `NO_BUNDLE`) that's greater than any valid version, e.g. `INT_MAX`)?

Reply:
Makes the table wider, but code simpler.
Force-pushed.